plugins/bright_plugin/src/bright-ui/src/app/constants/constants.ts

```diff
@@ -0,0 +1,13 @@
+import {Injectable} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class Constants {
+
+  public readonly baseApiUrl = "/ufmRestV2";
```

Comment:
do we need to hard-code this in the plugin? I think we should leave it to the UFM